docs/development/version-control/install-gitlab-on-ubuntu-18-04/index.md

```diff
@@ -113,4 +113,4 @@ nginx['redirect_http_to_https'] = true
 
 1. Navigate to your GitLab instance domain and verify that you are directed to `https`.
 
-  You are now ready to begin using GitLab as your remote version control system. Refer to [GitLab's official documentation](https://docs.gitlab.com/ee/README.html) for details on how to get started administering your GitLab instance.
```

Comment:
Looks like this line is identical, but why is it marked as diff?

Reply:
I see this happen sometimes but I don't know what it is.